static/js/zcommand.js

```diff
@@ -87,6 +88,31 @@ exports.process = function (message_content) {
         return true;
     }
 
+    if (tokens[0] === '/subscribe') {
+        tokens[0] = tokens[0].replace('/', '');
+
+        exports.send({
+            command: tokens.join(' '),
+            on_success: function (data) {
+                if (data.stream && data.user_id) {
+                    console.log(data.stream, data.user_id);
+                    if (!stream_data.is_user_subscribed(data.stream, data.user_id)) {
+                        stream_data.add_subscriber(data.stream, data.user_id);
+                        var msg = "Subcribed user to " + data.stream + "!";
+                    } else{
+                        var msg= "User is already subscribed!";
+                    }
+                    exports.tell_user(msg);
```

Comment:
Now that I'm reading this, it might be nicer if you just change `data.msg` and delete the `else` statement.